File: jackson/Docs&Code/L-TVAR/speed.m
N = 100;
StartN = 2;
NumTests = 10000;
averages = [];
for n = StartN:N
    A = rand(n, n);
    curr_times = [];
    for i = 1:NumTests
        tic;
        tmp = inv(A);
        curr_time = toc;
        curr_times{i} = curr_time;
    end
    averages{n-StartN+1} = mean(cell2mat(curr_times));
end

File: jackson/Docs&Code/L-TVAR/speed.R
library(matlib)

N = 100
StartN = 2
NumTests = 100
avgs = rep(0, N-StartN)
for (n in StartN:N) {
  A = matrix(rnorm(n*n), nrow=n)
  times = rep(0, NumTests)
  print(n)
  for (i in 1:NumTests) {
    start = Sys.time()
    tmp = inv(A)
    end = Sys.time()
    times[i] = end - start
  }
  avgs[n-StartN+1] = mean(times)
}